apps/dashboard/src/@3rdweb-sdk/react/hooks/useApi.ts

```diff
@@ -664,7 +664,7 @@ export function useCreateApiKey() {
 
   return useMutation(
     async (input: CreateKeyInput) => {
-      invariant(user?.address, "walletAddress is required");
+      // invariant(user?.address, "walletAddress is required");
 
```

Comment:
@jnsdls Is this intentional?

Reply:
I do not remember why I did this, however it SHOULD be safe because if the user is not logged in the call will simply fail?